File: C/Mid Codes/OddEven.c
//Find Odd or Even Number
#include<stdio.h>
int main(){
    int n;
    printf("Enter a number: ");
    scanf("%d",&n);
    if(n%2==0){
    printf("Even Number");
    }
    if(n%2!=0){
    printf("Odd number");
    }
    
return 0;
}

File: Java/Academic/Mid/OddEven.java
/*Write a Java program that takes an integer input from the user and checks whether the number is even or odd using an if-else statement.*/


import java.util.Scanner;

public class OddEven {
    public static void main(String[] args){
        Scanner in=new Scanner(System.in);
        System.out.println("Enter Number: ");
        int a=in.nextInt();
        if(a%2==0){
            System.out.println("Even Number");
        }
        else{
            System.out.println("Odd Number");

        }

    }
    
}
